public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx

```diff
@@ -55,10 +55,8 @@ export const TimeSeriesPanel: React.FC<TimeSeriesPanelProps> = ({
       <TooltipPlugin mode={options.tooltipOptions.mode} timeZone={timeZone} />
       <ZoomPlugin onZoom={onChangeTimeRange} />
       <ContextMenuPlugin timeZone={timeZone} replaceVariables={replaceVariables} />
-      {data.annotations ? (
+      {data.annotations && (
         <ExemplarsPlugin exemplars={data.annotations} timeZone={timeZone} getFieldLinks={getFieldLinks} />
-      ) : (
-        <></>
       )}
       {data.annotations ? <AnnotationsPlugin annotations={data.annotations} timeZone={timeZone} /> : <></>}
```

Comment:
```suggestion
      {data.annotations && (
        <AnnotationsPlugin annotations={data.annotations} timeZone={timeZone} />
      )}
```

Reply:
this is the current change

Reply:
ah, the same change or annotations plugin